source/Government.h

```diff
@@ -152,7 +154,7 @@ class Government {
 	const Phrase *hostileHail = nullptr;
 	const Phrase *hostileDisabledHail = nullptr;
 	std::string language;
-	const Fleet *raidFleet = nullptr;
+	std::vector<std::pair<const Fleet *, std::pair<double, double>>> raidFleets;
```

Comment:
It'd probably be cleaner at this point for Government to just have an inner class.
```
std::vector<RaidFleet> raidFleets;

class RaidFleet {
public:
	RaidFleet(const Fleet *fleet, double minAttraction, double maxAttraction);
	const Fleet *GetFleet() const;
	double MinAttraction() const;
	double MaxAttraction() const;

private:
	const Fleet *fleet = nullptr;
	double minAttraction;
	double maxAttraction;
}
```

Reply:
~~I suspected you'd ask this~~
Alr then